File: Programming Language(ver.2)/Python/402.py
str1 = input()
str2 = input()
n = int(input())
compare = 0

if n > len(str1):
    print('error')
    exit()

for i in range(n):
    compare += ord(str1[i]) - ord(str2[i])

if compare > 0:
    print(str1 + ' > ' + str2)
elif compare < 0:
    print(str1 + ' < ' + str2)
else:
    print(str1 + ' = ' + str2)

File: Programming Language(ver.2)/C/402.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main () 
{
 	char str1[128];
 	char str2[128];
	int n, result;
    scanf("%[^\n]%*c", str1);
    scanf("%[^\n]%*c", str2);
	scanf("%d", &n);
	
	if(n <= strlen(str1)){
		result = strncmp(str1, str2, n);
		if (result > 0) {
			printf("%s > %s\n", str1, str2);
		} else if (result == 0) {
			printf("%s = %s\n", str1, str2);
		} else {
			printf("%s < %s\n", str1, str2);
		}
	}
	else{
		printf("error\n");
	}
	return 0;
}
